addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -1808,4 +1808,30 @@ options.registry.CoverProperties = options.Class.extend({
         this.$target[0].dataset.filterColor = activeFilterColor || '';
     },
 });
+
+/**
+ * Allows snippets to be moved before the preceding element or after the following.
+ */
+options.registry.SnippetMove = options.Class.extend({
+
+    //--------------------------------------------------------------------------
+    // Options
+    //--------------------------------------------------------------------------
+
+    /**
+     * Moves the snippet around.
+     *
+     * @see this.selectClass for parameters
+     */
+    moveSnippet: function (previewMode, value, $opt) {
+        switch (value) {
+            case 'prev':
+                this.$target.prev().before(this.$target);
+                break;
+            case 'next':
+                this.$target.next().after(this.$target);
+                break;
```

Comment:
Finally there is no event to trigger when a move is performed ?

Reply:
Not that I know of, all the events that were triggered by the drag&drop were to update the UI to make the controls on the dragged element disappear, show drop zones, and transfer the left menu's control to the parent on drag start and back to the snippet on drag stop, which is unnecessary here since the move is instantaneous.